### PR TITLE
Desktop view

### PR DESCRIPTION
**Pull request by apanjain**

### CODE REVIEW

File: frontend/src/routes/Blog/DesktopView.js [hunk 1]
```diff
@@ -1,14 +1,46 @@
...
 class DesktopView extends Component {
     state = {  }
     render() { 
+        const blogData = this.props.blogData? this.props.blogData : null;
         return ( 
-            <React.Fragment>
-                <ComingSoon />
-            </React.Fragment>
+            <>
+            <div className={styles.container1}>
```
Comment (hacksdump) [inline frontend/src/routes/Blog/DesktopView.js]: Screenshot from 2020-02-19 13-23-49

Why the extra space between header and image?

File: frontend/src/routes/Events/DesktopView.js [hunk 2]
```diff
@@ -1,14 +1,47 @@
...
+    }
+    render() {
+        const eventData = this.props.eventData; 
         return ( 
-            <React.Fragment>
-                <ComingSoon />
-            </React.Fragment>
+            <>
+               <div className={styles.mainHeading}>We Conduct <span className='color-red'>Events</span> year-round</div>
+               <div className={styles.conatiner1}>
```
Comment (hacksdump) [inline frontend/src/routes/Events/DesktopView.js]: Typo in word container

File: frontend/src/routes/Events/DesktopView.js [hunk 3]
```diff
@@ -1,14 +1,47 @@
...
+                    <div className={styles.sectionHeading}>Upcoming Events</div>
+                    <ul className={styles.ul1}>
+                        <li><EventCard eventData={eventData} type='side'/></li>
+                        <li><EventCard eventData={eventData} type='side'/></li>
+                    </ul>
+               </div>
+               <div className={styles.container2}>
```
Comment (hacksdump) [inline frontend/src/routes/Events/DesktopView.js]: Screenshot from 2020-02-19 13-25-59
The images in the two sections should be left and right-aligned (parallel) to each other.

Comment (apanjain) [reply frontend/src/routes/Events/DesktopView.js]: done

File: frontend/src/routes/Blog/index.js [hunk 4]
```diff
@@ -15,28 +15,47 @@ const DesktopView = Loadable({
...
+        //         heading: value.title,
+        //         text: value.content,
+        //         metadata: {
+        //             d1: value.author,
+        //             d2: value.created_at,
+        //             d3: calculateReadingTime(value.content),
+        //         }
+        //     }
+        // })
+        const blogData =
```
Comment (rohitjha941) [inline frontend/src/routes/Blog/index.js]: API Integration

Check [axios](https://github.com/axios/axios)

File: frontend/src/routes/Events/MobileView.module.scss [hunk 5]
```diff
@@ -33,16 +33,16 @@
 }
 
 .eventCardCommon{
-    margin: 0 auto 0px auto;
+    margin: auto;
+    display: block;
 }
 
 .hr{
     margin: 20px auto 13px auto;
```
Comment (rohitjha941) [inline frontend/src/routes/Events/MobileView.module.scss]: Never use pixels

File: frontend/src/routes/Blog/index.js (hunk 4 above)
Comment (apanjain) [reply frontend/src/routes/Blog/index.js]: api integration will be done in another branch

**Pull request merged**